.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '30 3 * * 2'
```

Comment:
Could you add a short comment, why there is a scheduled build?

Reply:
Oh, that was copy-paste. I adjusted the schedule and added a comment.